INSTALL.md

```diff
@@ -50,7 +50,7 @@ to build a portable binary, add `PORTABLE=1` before your make commands, like thi
 ## Supported platforms
 
 * **Linux - Ubuntu**
-    * Upgrade your gcc to version at least 4.8 to get C++11 support.
+    * Upgrade your gcc to version at least 7 to get C++17 support.
```

Comment:
Should this mention clang?

Reply:
It's not the default compiler on Ubuntu

Reply:
Understood that it is not the default compiler.  Nowhere in INSTALL does it say we support clang (on any platform) and what the minimum clang version is.  Can we add that somewhere?

Reply:
It's near the top, "We do depend on newer gcc/clang with C++17 support (GCC >= 7, Clang >= 5)."